cpp/lib/include/FileUtil.h

```diff
@@ -191,6 +191,12 @@ class Directory {
 off_t GetFileSize(const std::string &path);
 
 
+/** \return The last modification timestamp of the file named by "path".
+ *  \note   Returns false if "path" does not exist or we don't have the rights to stat it.
+ */
+bool GetLastModificationDate(const std::string &path, timespec * mtim);
```

Comment:
- timespec contains a timestamp and not a date, so the function should rather be renamed to `GetLastModificationTimestamp`
- mtim should be changed to `timespec * const mtim`
- I'm not very familiar with the timespec data type, but it looks like the existing function GetMostRecentlyModifiedFile uses this data type as well. Maybe you could change that function to internally use the new function as well?